src/components/Header/Header.js

```diff
@@ -3,14 +3,15 @@ import { Link } from 'react-router-dom';
 import cx from 'classnames';
 
 import PageSettings from './PageSettings';
-import Button from '../shared/Button';
+import Button from '../Shared/Button';
```

Comment:
![image](https://user-images.githubusercontent.com/34597767/67608644-ce999100-f791-11e9-9ba6-c63ef8155478.png)

Reply:
I have renamed it back